README.md

```diff
@@ -82,6 +82,22 @@ This plugin contains the following features:
 - Ability to open the "Add and View comments" modal or "Update State" modal through buttons present in a notification post or a shared record post.
 - Supported record types for sharing a record - incident, problem, change_request, kb_knowledge, task, change_task and cert_follow_on_task.
 - Supported record types for updating a record state - incident, task, change_task and cert_follow_on_task.
+- Ability to open the incident modal through UI or slash command and create an incident in ServiceNow.
+- Ability to auto-subscribe to the newly created incident using a toggle switch present inside the incident modal.
```

Comment:
I saw the screenshot, and the label for toggle switch should be "Subscribe to the new incident". Please change that also in this PR.